datafusion/core/tests/sql/set_variable.rs

```diff
@@ -427,7 +427,10 @@ async fn set_time_zone_bad_time_zone_format() {
             .unwrap();
 
     let err = pretty_format_batches(&result).err().unwrap().to_string();
-    assert_eq!(err, "Parser error: Invalid timezone \"08:00\": only offset based timezones supported without chrono-tz feature");
+    assert_eq!(
```

Comment:
These changes are because the chrono-tz feature is now enabled for the common arrow version

This feature was enabled in https://github.com/apache/arrow-datafusion/pull/5603